test/unit/SwapOperator/closeOrder.js

```diff
@@ -127,36 +94,34 @@ describe('closeOrder', function () {
     await swapOperator.placeOrder(contractOrder, orderUID);
   });
 
-  describe('access control', function () {
-    it('before deadline, its callable only by controller', async function () {
-      const deadline = order.validTo;
-      const snapshot = await takeSnapshot();
+  it('before deadline, its callable only by controller', async function () {
+    const deadline = order.validTo;
+    const snapshot = await takeSnapshot();
```

Comment:
normally we have a beforeEach and afterEach to handle the snapshot encirclement

We should do the same here 

https://github.com/NexusMutual/smart-contracts/blob/862539ae93e7c00d5ccb9069f5afa0c5fcd4704e/test/unit/Assessment/index.js#L8